Traceback (most recent call last):
  File "/fs/homeu2/eccc/crd/ccrn_shr/reo000/MHW/MHW_Forecasts_New/MHW_Forecasts_New.py", line 154, in <module>
    find_MHW(il,years,qtile)
  File "/fs/homeu2/eccc/crd/ccrn_shr/reo000/MHW/MHW_Forecasts_New/MHW_Forecasts_New.py", line 143, in find_MHW
    for ikey in akeys:
TypeError: 'list' object is not callable
cleaning up temporary directories
